src/components/EntityPage/Participant/ParticipantClinical.js

```diff
@@ -198,7 +198,7 @@ class ParticipantClinical extends React.Component {
         accessor: 'shared_with',
         Cell: wrapper => {
 
-          if(wrapper.value === "0 participant") return <div>0 participant</div>;
+          if(wrapper.value === "0 participants") return <div>0 participant</div>;
```

Comment:
@claudia1296 Should the returned div also contain participant with an s at the end?

Reply:
@claudia1296 Since we have this twice in the code, we might as well make a tiny function to handle it.